### ISSUE TITLE
Clean up feature_update branch latest code

### ISSUE THREAD

**Issue by ZechenM** [ZechenM/bound-tolerance-research#3]
1. Specify Learning rate in TrainerArugment()
2. Allow specify hyperparamter on top of .py file.
3. Create seperate .py file (models.py, dataset.py) to increase readability. Include MNIST and CIFAR10/100/2 in dataset.py. Include (SimpleModel, myResNet, VGG-13/16 in models.py) (Edit: VGG-13 or 16 model can be directly imported from HuggingFace Transformer library, no need to store them in models.py. Only store custom models.)
